.flake8

```
@@ -89,6 +89,11 @@ per-file-ignores =
   # have <= members in tests (WPS202), including many local vars (WPS210),
   # additionally test docstrings don't need param lists (DAR, DCO020):
   tests/**.py: DAR, DCO020, S101, S404, S603, WPS202, WPS210, WPS436, WPS442, WPS450
```

let's add a separator line.
```suggestion
  tests/**.py: DAR, DCO020, S101, S404, S603, WPS202, WPS210, WPS436, WPS442, WPS450

```